### ISSUE TITLE
Fultoning to an unloading beacon does unexpected things

### ISSUE THREAD

**Issue by extramrdo** [FTL13/FTL13#3646]
## Testmerges: #3639: 
'New world topic for getting the current list of players in discord' by crossedfall at commit c5b7c8

## ~~Reproduction:~~ Today in weird edge cases:
"alright, so I jumped into the FTL stream to fulton myself to a ruin we found in orbit at the last planet we were actively leaving, did the fulton, and now my viewpoint is stuck in space where I was. I can interact with all the items in my inventory but I don't have a self to use them on, and I can't move even though I have a jetpack ON on my back"

Plant fulton beacon on ship. Visit In Orbit. Plant fulton beacon on In Orbit z-level, in a ruin or wherever. Acquire Fulton Pack. **Do not use fulton-pack in hand yet.** Return to ship via FOB or whatever. Engage in FTL to other star system. Wear a space suit. During FTL transit, exit ship. 
Fall into Space. Background goes from moving space to stationary space. Pull fulton pack from backpack. Use in hand. List of beacons to sync fulton pack to pops up.

**Expected result**: There is one fulton beacon in the list, the shipside one.
**_Observed_ result**: There are two fulton beacons in the list, the shipside one and the one in the z-level ruin.

Choose the ruin beacon. Click on self with the fulton pack.

**Expected result**: The fulton beacon fails to take you, because the ruin's z-level is unloaded.
**Expected result**: The fulton beacon successfully takes you to the ruin. The fulton pack enters your backpack with 1 less charge.
**Expected result**: You are unloaded before this can happen, and become a ghost because you are dead.
**_Observed_ result**: The fulton beacon does the balloon animation, leaving the viewpoint exactly where it was. Your sprite, as part of the animation, vanishes, and does not return. The fulton pack enters your backpack with one less charge as normal. You can interact with items in your backpack and body, including removing a backpack and replacing it with your jet harness. Attempting to then use the arrow keys to fly around yields no change in state; the background remains stationary, no jet particles appear, etc.

Wait. Ship finally finishes FTL transition and the round ends.

Expected result: Since the ship has finally changed to a new system, your z-level should have been unloaded one way or another. Y'ought to be a ghost or dead or something.
Observed result: Still in the bodiless space state.

In short... **Bug**: Played stupid game. Won stupid prizes.

**Comment by crossedfall** [comment]
I tried to look into this recently but wasn't able to reproduce it, it seems like one of those fringe annoyances. I would say to ahelp next time it happens and we can port you back to the ship safely. I am going to close this issue since it doesn't seem like something that can be easily/quickly fixed.
